File: app/src/main/java/com/productions/gizzmoo/pokemonpuzzleleague/settings/SettingsActivity.java
package com.productions.gizzmoo.pokemonpuzzleleague.settings;

import android.app.Activity;
import android.content.Context;
import android.content.SharedPreferences;
import android.os.Bundle;

/**
 * Created by Chrystian on 4/7/2018.
 */

public class SettingsActivity extends Activity {
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        // Display the fragment as the main content.
        getFragmentManager().beginTransaction().replace(android.R.id.content, new SettingsFragment()).commit();
        SharedPreferences settings = getApplicationContext().getSharedPreferences("PreferencesName", Context.MODE_PRIVATE);
        settings.edit().clear().apply();
    }
}

File: app/src/main/java/com/productions/gizzmoo/pokemonpuzzleleague/settings/SettingsActivity.kt
package com.productions.gizzmoo.pokemonpuzzleleague.settings

import android.content.Context
import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class SettingsActivity : AppCompatActivity() {
    public override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        // Display the fragment as the main content.
        supportFragmentManager.beginTransaction().replace(android.R.id.content, SettingsFragment()).commit()
        val settings = applicationContext.getSharedPreferences("PreferencesName", Context.MODE_PRIVATE)
        settings.edit().clear().apply()
    }
}